backend/src/v2/metadata/converter.go

```diff
@@ -177,13 +181,20 @@ func toRuntimeArtifact(artifact *pb.Artifact) (*pipelinespec.RuntimeArtifact, er
 	errorF := func(err error) (*pipelinespec.RuntimeArtifact, error) {
 		return nil, fmt.Errorf("failed to convert MLMD artifact to RuntimeArtifact: %w", err)
 	}
+	// Retrieve custom_path value from artifact.CustomProperties, if present.
+	var customPathStr string
+	if artifact.CustomProperties != nil {
+		customPath := artifact.CustomProperties["custom_path"]
```

Comment:
The custom path flow should be:
1. Executor sets the custom path property on the output artifact in the executor output
2. The `uploadOutputArtifacts` function checks if the custom path is present and uses that to upload instead of the predefined path
3. The custom path is no longer needed and be ignored and not set on the MLMD artifact.